packages/swap/src/routes/__tests__/__snapshots__/quote.test.ts.snap

```diff
@@ -40,6 +40,90 @@ exports[`server GET /quote gets the quote for deprecated params without the chai
 }
 `;
 
+exports[`server GET /quote gets the quote from btc with boost information 1`] = `
+{
+  "boostedQuote": {
```

Comment:
not sure if we use `boosted` somewhere, could also call it `boostQuote` to keep the feature name?

Reply:
Will do